### PR TITLE
docs: modifying volumes guide to remove ambiguity. 

### PR DESCRIPTION
**Pull request by ghost**
Having dbdata used for both volume and container is quite confusing to learners. Proposing forelisted change. Thank you

### CODE REVIEW

File: docs/userguide/dockervolumes.md [hunk 1]
```diff
@@ -260,11 +260,11 @@ we'll be left with a backup of our `dbdata` volume.
 You could then restore it to the same container, or another that you've made
 elsewhere. Create a new container.
 
-    $ docker run -v /dbdata --name dbdata2 ubuntu /bin/bash
+    $ docker run -v /dbdata --name dbstore ubuntu /bin/bash
```
Comment (thaJeztah) [inline docs/userguide/dockervolumes.md]: This should probably be "dbstore2" here

File: docs/userguide/dockervolumes.md [hunk 2]
```diff
@@ -260,11 +260,11 @@ we'll be left with a backup of our `dbdata` volume.
...
 
 Then un-tar the backup file in the new container's data volume.
 
-    $ docker run --volumes-from dbdata2 -v $(pwd):/backup ubuntu cd /dbdata && tar xvf /backup/backup.tar
+    $ docker run --volumes-from dbstore -v $(pwd):/backup ubuntu cd /dbdata && tar xvf /backup/backup.tar
```
Comment (thaJeztah) [inline docs/userguide/dockervolumes.md]: Same here

**Comment by thaJeztah** [comment]
Thanks, added some comments inline.

(We may want to rewrite this article now that we have named volume features and volume management)

/cc @cpuguy83

**Comment by tonov** [comment]
I've made corrections as per your recommendation, please review. thank youo

**Comment by thaJeztah** [comment]
Thanks! 

LGTM, but can you "squash" your commits so that there's only one commit in this PR?

ping @moxiegirl for review

**Comment by moxiegirl** [comment]
@tonov Thank you! LGTM

**Comment by thaJeztah** [comment]
@tonov if you can squash your commits, we can merge. If you need help doing that, let us know as well (or ask for assistance in the #docker-dev IRC channel) :+1:

**Comment by LK4D4** [comment]
ping @tonov pls squash

**Comment by moxiegirl** [comment]
See #18016 for the carry

**Pull request closed without merging**